src/main/java/it/pagopa/interop/probing/eservice/registry/reader/producer/ServicesSend.java

```diff
@@ -9,12 +9,13 @@
 import it.pagopa.interop.probing.eservice.registry.reader.config.aws.sqs.SqsConfig;
 import it.pagopa.interop.probing.eservice.registry.reader.config.jacksonmapper.JacksonMapperConfig;
 import it.pagopa.interop.probing.eservice.registry.reader.dto.EserviceDTO;
-import it.pagopa.interop.probing.eservice.registry.reader.util.logging.LoggingMessages;
-import lombok.extern.slf4j.Slf4j;
+import it.pagopa.interop.probing.eservice.registry.reader.util.logging.Logger;
+import it.pagopa.interop.probing.eservice.registry.reader.util.logging.impl.LoggerImpl;
+
 
-@Slf4j
 public class ServicesSend {
 
+  private final Logger logger = new LoggerImpl();
```

Comment:
If possibile, also in the project without DI, use a DI library and use `@Inject` , so projects are similar each other and all using configuration for producers. Also in another step.